packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -191,6 +206,16 @@ - (AVCaptureConnection *)createConnection:(NSError **)error {
     connection.videoMirrored = YES;
   }
 
+  if (_fps) {
+    [_videoCaptureSession beginConfiguration];
+    NSError *outError;
+    [_captureDevice lockForConfiguration:&outError];
+    _captureDevice.activeVideoMinFrameDuration = CMTimeMake(1, [_fps intValue]);
+    _captureDevice.activeVideoMaxFrameDuration = CMTimeMake(1, [_fps intValue]);
```

Comment:
why do we set the same fps for both? should dart side send the min/max instead?

Reply:
Actually set min and max to same value enforces constant fps of recorded video. Non constant fps videos are have bad behavior in most conditions, causing freezed and jumping video artifacts. So i decided to limit users for constant-only fps.

Reply:
Can you add a comment explaining about this?